keanu-project/src/main/java/io/improbable/keanu/network/ProtobufLoader.java

```diff
@@ -213,13 +205,39 @@ private Vertex instantiateVertex(Class vertexClass,
         }
     }
 
+    private Object getParameter(Parameter methodParameter,
+                                Map<String, Object> paramMap,
+                                KeanuSavedBayesNet.Vertex vertex) {
+        LoadVertexParam paramAnnotation;
+        LoadShape shapeAnnotation;
+
+        if ((paramAnnotation = methodParameter.getAnnotation(LoadVertexParam.class)) != null) {
+            Object parameter = paramMap.get(paramAnnotation.value());
+            if (parameter == null) {
+                throw new IllegalArgumentException("Failed to create vertex due to missing parent: "
+                    + paramAnnotation.value());
+            }
```

Comment:
for consistency (with line 224) I think you should put an `else` here

Reply:
Fixed